MovieGallery/Pages/Search.cshtml.cs

```diff
@@ -22,16 +23,17 @@ public class SearchModel : PageModel
 
         public void OnGet(string SearchbyYear)
         {
+            string currentYear = DateTime.Now.Year.ToString();
             using (StreamReader r = new StreamReader("Movies.json"))
             {
-                string json = r.ReadToEnd();
-                MoviesJson = JsonConvert.DeserializeObject<Movies>(json);
+                string ReadMoviejson = r.ReadToEnd();
+                MoviesJson = JsonConvert.DeserializeObject<Movies>(ReadMoviejson);
             }
 
             using (StreamReader r = new StreamReader("Shows.json"))
             {
-                string json = r.ReadToEnd();
-                ShowsJson = JsonConvert.DeserializeObject<Shows>(json);
+                string ReadShowjson = r.ReadToEnd();
+                ShowsJson = JsonConvert.DeserializeObject<Shows>(ReadShowjson);
```

Comment:
Variable name has been changed here for better understanding. Also same variable name was being used to read both movie and show json. To remove duplicate, variable name has been changed.